transaction/errors_test.go

```diff
@@ -45,15 +46,15 @@ func TestBadRequestError(t *testing.T) {
 func TestErrorLogEncoder(t *testing.T) {
 	tcs := []struct {
 		desc      string
-		getLogger GetLoggerFunc
+		getLogger sallust.GetLoggerFunc
```

Comment:
sallust shouldn't export a function type for this.  It requires apps to express a dependency in many cases just to pull in this type.  Using func(context.Context) *zap.Logger as the type suffices.

Reply:
my bad, thought I removed all these references. Double checking other servers as well